src/test/resources/multitenancy/roles.yml

```diff
@@ -2,6 +2,8 @@
 _meta:
   type: "roles"
   config_version: 2
+security_rest_api_access:
```

Comment:
For anyone wondering what's special about this role and how it gets restapi access, its one of the roles specified in the demo configuration for restapi access here: https://github.com/opensearch-project/security/blob/main/tools/install_demo_configuration.sh#L384

Reply:
A small addition from my side.  
There are 3 types of admins in the plugin:
- `super admin` - has access to everything e.g. can change configuration like SSL reload, update config on each node,   CRUD operations for nodes ds etc. This is leftovers from the past idea to have a dedicated node admin  
- `admin` - has full access to cluster and indices
- `REST admin API` - "admin" which has access to change some plugin configuration like roles, roles mapping, internal users etc.

compare to other security frameworks the plugin idea is very very confusing and it is hard to support such config.